src/transformers/integrations/integration_utils.py

```diff
@@ -1543,7 +1653,40 @@ def on_save(self, args, state, control, **kwargs):
             ckpt_dir = f"checkpoint-{state.global_step}"
             artifact_path = os.path.join(args.output_dir, ckpt_dir)
             logger.info(f"Logging checkpoint artifacts in {ckpt_dir}. This may take time.")
```

Comment:
This message should be update to reflect saving under `name`

Reply:
done